src/modelmanager.cpp

```diff
@@ -828,6 +839,41 @@ void ModelManager::watcher(std::future<void> exitSignal) {
     SPDLOG_LOGGER_INFO(modelmanager_logger, "Stopped model manager thread");
 }
 
+void ModelManager::cleanerRoutine(uint32_t resourcesCleanupIntervalSec, uint32_t sequenceCleanerIntervalMinutes, std::future<void> cleanerExitSignal) {
+    SPDLOG_LOGGER_INFO(modelmanager_logger, "Started cleaner thread");
+
+    uint32_t resourcesWaitTime = resourcesCleanupIntervalSec;
+    uint32_t sequenceWaitTime = sequenceCleanerIntervalMinutes * 60;
+    uint32_t currentWaitTime = (resourcesWaitTime < sequenceWaitTime || sequenceWaitTime == 0) ? resourcesWaitTime : sequenceWaitTime;
+    bool shouldCleanupSequence = sequenceCleanerIntervalMinutes != 0;
```

Comment:
```suggestion
    bool shouldCleanupSequence = sequenceCleanerIntervalMinutes != 0;
    uint32_t currentWaitTime = (!shouldCleanupSequence || resourcesWaitTime < sequenceWaitTime) ? resourcesWaitTime : sequenceWaitTime;
```
This way you first tell in first line what is the meaning of 0 and then avoid comparison. Then order in the second line matters as well since if you know it is 0 then we shouldn;t even compare it to resourceWaitTime